platform/src/main/java/org/conscrypt/TrustManagerImpl.java

```diff
@@ -420,13 +420,13 @@ public void handleTrustStorageUpdate() {
 
     private byte[] getOcspDataFromSession(SSLSession session) {
         List<byte[]> ocspResponses = null;
-        if (session instanceof AbstractOpenSSLSession) {
-            AbstractOpenSSLSession opensslSession = (AbstractOpenSSLSession) session;
-            ocspResponses = opensslSession.getStatusResponses();
+        if (session instanceof ActiveSession) {
+            ActiveSession opensslSession = (ActiveSession) session;
+            ocspResponses = opensslSession.getPeerOcspStapledResponse();
         } else {
             Method m_getResponses;
             try {
-                m_getResponses = session.getClass().getDeclaredMethod("getStatusResponses");
+                m_getResponses = session.getClass().getDeclaredMethod("getOcspStapledResponse");
```

Comment:
This probably shouldn't be renamed since it's the [name of record](http://download.java.net/java/jdk9/docs/api/javax/net/ssl/ExtendedSSLSession.html#getStatusResponses--) currently.